sqlglot/dialects/dialect.py

```diff
@@ -670,9 +670,9 @@ class Dialect(metaclass=_Dialect):
         "DEC": "DECADE",
         "DECS": "DECADE",
         "DECADES": "DECADE",
-        "MIL": "MILLENIUM",
-        "MILS": "MILLENIUM",
-        "MILLENIA": "MILLENIUM",
+        "MIL": "MILLENNIUM",
+        "MILS": "MILLENNIUM",
+        "MILLENIA": "MILLENNIUM",
```

Comment:
Nice catch, the previous forms would also result in invalid SQL if the input was abbreviated Postgres.